Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Yul Optimizer: Fix reordering bug in connection with shift instructions in for loop conditions.
```

Comment:
Edit: In connection with division or multiplication combined with right/left shifted one in for loop conditions?

Reply:
Also, did we decide to skip a bug list entry for this?